src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -0,0 +1,298 @@
+          : height,
+      });
+      setBoxes(
+        update(boxes, {
+          [id]: {
+            $merge: { width, height },
+          },
+        })
+      );
+    },
+    [
+      boxes,
+      coordinatesBoundary.height,
+      coordinatesBoundary.width,
+      imageDims,
+      onResize,
+      resizable,
+    ]
+  );
+
+  const [, drop] = useDrop({
+    accept: ItemTypes.SUBVENUE_ICON,
+    drop(item: DragItem, monitor) {
+      if (!interactive) return;
+      const delta = monitor.getDifferenceFromInitialOffset() as {
+        x: number;
+        y: number;
+      };
+
+      let left = Math.round(item.left + delta.x);
+      let top = Math.round(item.top + delta.y);
+      if (snapToGrid) {
+        [left, top] = doSnapToGrid(left, top);
+      }
+
+      moveBox(item.id, left, top);
+    },
+  });
+
+  const renderSelectedRoom = (index: number) => {
+    return (
+      <>
+        <DraggableSubvenue
+          isResizable={resizable}
+          id={index.toString()}
+          imageStyle={iconImageStyle}
+          rounded={!!rounded}
+          {...boxes[index]}
+          onChangeSize={resizeBox(index.toString())}
+          lockAspectRatio={lockAspectRatio}
+        />
+        {imageDims && interactive && (
+          <CustomDragLayer
+            snapToGrid={!!snapToGrid}
+            rounded={!!rounded}
+            iconSize={boxes[Object.keys(boxes)[index]]}
+          />
+        )}
+      </>
+    );
+  };
+
+  return (
+    <div ref={drop} style={{ ...styles, ...containerStyle }}>
+      <ReactResizeDetector
+        handleWidth
+        handleHeight
+        onResize={(width, height) => setImageDims({ width, height })}
+      />
+      <img
+        alt="draggable background "
+        style={{
+          width: "100%",
+          ...backgroundImageStyle,
+        }}
+        src={backgroundImage}
+      />
+
+      {backgroundImage &&
+        rooms.map((room, index) =>
+          room === selectedRoom ? (
+            <div key={`${room.title}-${index}`}>
+              {renderSelectedRoom(index)}
+            </div>
+          ) : (
+            <div
+              className={"Container__room-preview"}
+              style={{
+                top: `${room.y_percent}%`,
+                left: `${room.x_percent}%`,
+                width: `${room.width_percent}%`,
+                height: `${room.height_percent}%`,
+              }}
+              key={`${room.title}-${index}`}
+              onClick={() => !selectedRoom && setSelectedRoom(room)}
+            >
+              <img
+                className={classNames("Container__room-image", {
+                  "Container__room-image--disabled": !room.isEnabled,
+                })}
+                src={room.image_url}
+                alt="room-logo"
+                title={room.title}
+              />
+              <div className="Container__room-title">{room.title}</div>
+            </div>
+          )
+        )}
+    </div>
```

Comment:
into rendered

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/e3b301bfa611c387d3e2ac24b7195994415ffc70